tests/FSharp.Test.Utilities/ScriptHelpers.fs

```diff
@@ -41,10 +41,14 @@ type private EventedTextWriter() =
             sw.WriteLine line
             lineWritten.Trigger(line)
         else sb.Append(c) |> ignore
-    override _.ToString() = sw.ToString()
+    override _.ToString() =
+        sw.Flush()
+        sw.ToString()
 
 type FSharpScript(?additionalArgs: string[], ?quiet: bool, ?langVersion: LangVersion, ?input: string) =
 
+    do ignore input
```

Comment:
This was only used in one test that is temporarily skipped awaiting a rewrite.

Reply:
Wait so cannot this just be removed then? :)